test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs

```diff
@@ -3,842 +3,842 @@
 
 namespace Microsoft.TestPlatform.Utilities.UnitTests
 {
-    using System;
```

Comment:
what is going on here? were there tabs instead of spaces?

Reply:
hard to see from this view but the file has few thousand changes.

Reply:
Yes so i used the [FixMixedTabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs) plugin to remove spaces with tabs, so looks like the plugin rewrites the whole cs file.